dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubHelper.java

```diff
@@ -42,10 +47,50 @@
 public final class GitHubHelper {
 
     private static final Logger LOG = LoggerFactory.getLogger(GitHubHelper.class);
+    public static final String BRANCHNAME_FOR_BOT = "feature/add_dockstore_yml";
+
+    public static final String DOCKSTORE_BOT_PR_TEXT = """
+           Dockstore's bot has guessed at a .dockstore.yml for your repository.
```

Comment:
Could tweak this text to help users understand what Dockstore did, what it found, etc.  For example:
```
Dockstore inspected your repository and found the following workflows, tools, and notebooks:
[list]
```

IMHO, "bot" is rather colloquial and not entirely accurate, we could just say that "Dockstore" did it...

Reply:
Will add a ticket for the former (seems rather open-ended)

For the second, I think it's fairly common e.g.

- https://github.com/dependabot
- https://github.com/renovatebot
- https://github.com/googleapis/repo-automation-bots

also shows up in GitHub documentation

> Bot accounts are specific to GitHub Apps and are built into every GitHub App.

https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/differences-between-github-apps-and-oauth-apps#machine-vs-bot-accounts

> Managing bots and service accounts with two-factor authentication

https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-two-factor-authentication-for-your-organization/managing-bots-and-service-accounts-with-two-factor-authentication